### PR TITLE
Create an Issue Finder page on the site

### PR DESCRIPTION
**Pull request by dhruvkb**
## Description
This PR adds a new 'Issue Finder' page to the site.

## Screenshot
2020-09-06 13 12 19 localhost 8f95021b0a61

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by dhruvkb** [comment]
@kgodey apparently the GitHub API required [authentication](https://developer.github.com/v3/issues/#list-organization-issues-assigned-to-the-authenticated-user). Can we generate a read-only no-permissions token for use on the site?

**Comment by Dhruvi16** [comment]
This is awesome. @dhruvkb 👍🏼

File: webpack/js/issue-finder.js [hunk 1]
```diff
@@ -0,0 +1,269 @@
+import {Octokit} from '@octokit/rest';
+import Vue from 'vue';
+
+import VueSelect from 'vue-select';
+
+const octokit = new Octokit({
+  auth: ''
```
Comment (dhruvkb) [inline webpack/js/issue-finder.js:7]: This is required.

**Review by kgodey — CHANGES_REQUESTED** [review]
@dhruvkb this is a great start! some high level feedback:
- please make the filters all on one line so that they don't take up the entire page – right now, you can't even see the issues unless you scroll. alternatively, use sidebar filters like CC Search
- let's remove the aspect filter, I don't see it getting a lot of use.
- I think we should change the experienced filter to "is this your first time contributing to CC" and then filter by "good first issue", a yes/no question is an easier choice.
- can we populate the skills based on the technologies used in the cc metadata file? it seems better than having skills/technologies in two different places (that isn't related to this PR exactly, so feel free to make a different issue for it)
- can we have background colors for the labels?

**Comment by dhruvkb** [comment]
Updated screenshot:
2020-09-10 18 53 27 localhost b32efca54dba

**Review by hugosolar — APPROVED** [review]
This looks awesome, I think it will be very helpful, especially for internships. Great work @dhruvkb

**Review by aldenstpage — APPROVED** [review]

**Review by kgodey — CHANGES_REQUESTED** [review]
@dhruvkb this looks fantastic! a couple more small things:
- it looks like the filtering is happening instantly for skills, but the experience level requires pressing the search button. please standardize on one or the other and remove the search button if needed.
- please add some introductory text to the top of the page explaining that these are all issues that we are looking for help with, etc.
- please link to the issue finder from the contribution guidelines

**Review by zackkrida — CHANGES_REQUESTED** [review]
_Love_ the redesign. One suggestion—adding `placeholder="Any skills"` to the `#skills` VueSelect.

Screen Shot 2020-09-10 at 7 20 41 AM

**Review by zackkrida — APPROVED** [review]

**Comment by kgodey** [comment]
@dhruvkb the changes look good! I'll re-review this after the new databag implementation.

**Comment by dhruvkb** [comment]
Databag implementation creativecommons/ccos-scripts#79

**Review by kgodey — APPROVED** [review]

**Pull request merged**